src/patternfly/components/AlertGroup/alert-group.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../patternfly-utilities";
+
+// Tabs
+.pf-c-alert-group {
+  // Alert Group variables
+  --pf-c-alert-group__item--MarginBottom: var(--pf-global--spacer--sm);
+
+  // Toast variables
+  --pf-c-alert-group--m-toast--Top: 0;
+  --pf-c-alert-group--m-toast--Right: 0;
+  --pf-c-alert-group--m-toast--MaxWidth: #{pf-size-prem(600px)};
+  --pf-c-alert-group--m-toast--PaddingTop: var(--pf-global--spacer--2xl);
+  --pf-c-alert-group--m-toast--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-alert-group--m-toast--PaddingBottom: 0;
+  --pf-c-alert-group--m-toast--PaddingLeft: var(--pf-global--spacer--xl);
+
+  width: 100%;
+
+  // Spacing between alerts
+  > .pf-c-alert:not(:last-child) {
+    margin-bottom: var(--pf-c-alert-group__item--MarginBottom);
+  }
+
+  // Toast positioning modifier
+  &.pf-m-toast {
+    position: fixed;
+    top: var(--pf-c-alert-group--m-toast--Top);
+    right: var(--pf-c-alert-group--m-toast--Right);
+    max-width: var(--pf-c-alert-group--m-toast--MaxWidth);
+    padding: var(--pf-c-alert-group--m-toast--PaddingTop) var(--pf-c-alert-group--m-toast--PaddingRight) var(--pf-c-alert-group--m-toast--PaddingBottom) var(--pf-c-alert-group--m-toast--PaddingLeft);
+    pointer-events: none;
```

Comment:
this is 🔥🔥🔥

Reply:
Your suggestion below was more 🔥 and allowed us to cut 11 lines of CSS!